src/main/java/in/projecteka/consentmanager/consent/ConsentNotificationPublisher.java

```diff
@@ -53,9 +55,9 @@ private void broadcastArtefactsToHips(ConsentArtefactsMessage message) {
         message.getConsentArtefacts()
                 .forEach(consentArtefact -> {
                     sendMessage(consentArtefact, destinationInfo.getExchange(), destinationInfo.getRoutingKey());
-                    logger.info(String.format(
-                            "Broadcasting consent artefact notification to hip for consent artefact: %s",
-                            consentArtefact.getConsentDetail().getConsentId()));
+                    logger.info(
```

Comment:
I think sonar fails because of string append

Reply:
Fixed